presto-spi/src/main/java/com/facebook/presto/spi/ConnectorTableMetadata.java

```diff
@@ -18,32 +18,38 @@
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
 import static java.util.Collections.emptyMap;
 
 public class ConnectorTableMetadata
 {
     private final SchemaTableName table;
+    private final Optional<String> comment;
     private final List<ColumnMetadata> columns;
     private final Map<String, Object> properties;
 
     public ConnectorTableMetadata(SchemaTableName table, List<ColumnMetadata> columns)
     {
-        this(table, columns, emptyMap());
+        this(table, columns, emptyMap(), Optional.empty());
     }
 
-    public ConnectorTableMetadata(SchemaTableName table, List<ColumnMetadata> columns, Map<String, Object> properties)
+    public ConnectorTableMetadata(SchemaTableName table, List<ColumnMetadata> columns, Map<String, Object> properties, Optional<String> comment)
```

Comment:
Add another constructor so that we don't have to update all call sites and more importantly, don't break existing connectors (we avoid incompatible SPI changes when possible). 

Please remove all non-required call site changes, as this makes the diff easier to read.